### PR TITLE
feat: s2 foundation data set

### PR DESCRIPTION
**Pull request by mrcjhicks**
## Description
Initial S2 Foundation data set synced with[ spectrum-tokens/snapshot-s2-foundations](https://github.com/adobe/spectrum-tokens/tree/snapshot-s2-foundations)

## Types of changes

<li> [ ] Patch (bug fixes, typos, mistakes; non-breaking change which fixes an issue) </li>
<li> [ ] Minor (add a new token, changing a value, deprecating a token; non-breaking change which adds functionality) </li>
<li> [ ] Major (deleting a token, changing token value type, renaming a token by deprecating the old one; fix or feature that would cause existing functionality to change) </li>

## Checklist:

<li> [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html). </li>
<li> [ ] I updated the token in all applicable sets. This applies if updating, adding, or deleting a token that has data across different sets (for example, if the value differs across color themes.) </li>

### CODE REVIEW

**Comment by mrcjhicks** [comment]
**[TESTING foundations](https://github.com/adobe/spectrum-tokens-studio-data/actions/runs/10583825113)**

READING spectrum-tokens: `/spectrum-tokens/packages/tokens/src/`
READING   tokens studio: `/spectrum-tokens-studio-data/src/tokens-studio/foundations/`

TEST COMPLETE

**Comment by mrcjhicks** [comment]
**[TESTING foundations](https://github.com/adobe/spectrum-tokens-studio-data/actions/runs/10583882388)**

READING spectrum-tokens: `/spectrum-tokens/packages/tokens/src/`
READING   tokens studio: `/spectrum-tokens-studio-data/src/tokens-studio/foundations/`

TEST COMPLETE

**Review by karstens — APPROVED** [review]

File: src/tokens-studio/foundations/$themes.json [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+    "name": "dark",
+    "selectedTokenSets": {
+      "spectrum/opacity.alias/dark": "enabled",
+      "spectrum/opacity.component/dark": "enabled"
+    },
+    "group": "S.Color-theme"
+  },
+  {
+    "id": "09fb335cab18186b281e7d5394bf56a0fa6e625f",
+    "name": "darkest",
```
Comment (karstens) [inline src/tokens-studio/foundations/$themes.json:81]: @mrcjhicks maybe this slipped through?

Comment (GarthDB) [reply src/tokens-studio/foundations/$themes.json:81]: I'll search and see if any others slipped through.

Comment (GarthDB) [reply src/tokens-studio/foundations/$themes.json]: Looks like that was the only place darkest was included, and it wasn't in any actual data, just this metadata.

**Comment by mrcjhicks** [comment]
**[TESTING foundations](https://github.com/adobe/spectrum-tokens-studio-data/actions/runs/10601143364)**

READING spectrum-tokens: `/spectrum-tokens/packages/tokens/src/`
READING   tokens studio: `/spectrum-tokens-studio-data/src/tokens-studio/foundations/`

TEST COMPLETE

**Review by karstens — APPROVED** [review]

**Pull request merged**